### ISSUE TITLE
doesn't work: The termshark UI will start when packets are detected on eth0...

### ISSUE THREAD

**Issue by 532910** [gcla/termshark#144]
## Prerequisites

Please verify these before submitting an issue.

- [x] I am running the latest versions of Termshark and Wireshark.
- [x] I checked the [README](https://github.com/gcla/termshark) and [User Guide](https://github.com/gcla/termshark/blob/master/docs/UserGuide.md) and found no answer
- [x] I searched [issues](https://github.com/gcla/termshark/issues?q=is%3Aissue) and this has not yet been filed

## Problem 
Doesn't work.

### Current Behavior
```
# termshark
(The termshark UI will start when packets are detected on eth0...)
```

Please describe the behavior you are seeing. 

### Expected Behavior

UI should start, eth0 is the main interface with a lot of packets.

Please describe the behavior you are expecting. 

### Screenshots as applicable 

* Provide screenshots of wireshark/tshark if that's what the expectation is based on.
* Provide screenshots of the problem state.

## Context

Please provide the complete output of these commands:

* termshark -v (or termshark -vv if running from git/HEAD)
* termshark -v | cat

```
# termshark -vv 
termshark 2.4.0
using tshark 4.0.2 (from /usr/bin/tshark)
```

Please also provide any relevant information about your environment (OS, VM, pi,...)
debian sid

**Comment by pocc** [comment]
Questions for you:

* Do you see packets when using wireshark or tshark on eth0? 
* Are you able to replicate this behavior on other systems?

You can run this command to see what traffic is being sent on various interfaces in linux (provided you have ripgrep and ethtool installed):

```
ifaces=$(ifconfig | rg '^(\w+):.*' -r '$1');
for iface in $ifaces;
  do echo "=== ${iface} ===";
  ethtool -S $iface;
done
```

**Comment by 532910** [comment]
Yes, I see packets with wireshark, tshark and tcpdump. No, I have no other systems.

**Comment by zrr1999** [comment]
我也遇到了这个问题，https://github.com/gcla/termshark/blob/master/docs/FAQ.md#how-can-i-make-termshark-run-without-root 按这个改了一下就可以了，我的看日志应该就是权限的问题。

**Comment by 532910** [comment]
`#` means I run it as root
